mlflow/tracing/fluent.py

```diff
@@ -141,7 +144,7 @@ def get_traces(n: int = 1) -> List:
 
 
 # NB: Extracting as a function just to allow mocking in tests
-def _get_tracer(module_name: str):
+def _get_tracer():
     from mlflow.tracing.provider import get_tracer
 
-    return get_tracer(module_name)
```

Comment:
note: It seems the module name passed to the tracer is not really important when using a single tracer instance throughout the application.